packages/web/utils/playerHelpers.ts

```diff
@@ -57,6 +57,27 @@ export const getGuildCoverImageSmall = (): string => GuildCoverImageSmall.src;
 export const getPlayerUsername = (player?: Maybe<Player>): string | undefined =>
   formatIfAddress(player?.profile?.username ?? undefined);
 
+export const getPlayerURL = (
+  player?: Maybe<Player | GuildPlayer>,
+  opts: { rel?: boolean; default?: string } = {},
+): string | undefined => {
```

Comment:
This can probably be inferred.

```suggestion
) => {
```